### ISSUE TITLE
Provide Way to back Up and Restore Auto Queue Preferences

### ISSUE THREAD

**Issue by eroc1990** [podverse/podverse-rn#1890]
**Is your feature request related to a problem? Please describe.**
If/when I set up a new device, or have to remove and reinstall the app, I need to reconfigure my auto-queue items. 

**Describe the solution you'd like**
Provide the ability to back up and restore auto queue behavior, either by local export or cloud sync linked to my Podverse account. 

**Describe alternatives you've considered**
Right now, the only alternative is manually reconfiguring, which while not horrible, is time consuming and feels a bit cumbersome.

**Comment by mitchdowney** [comment]
@eroc1990 yeah sorry...the local settings for the mobile app do not currently get saved to our server and sync across other devices. This would definitely be nice to have. We would probably complete it however as part of a larger initiative to handle syncing many settings with server, including these two.

**Comment by eroc1990** [comment]
I get it 100%. Here's hoping that day comes, but that won't stop me from using the app in the meantime.

**Comment by mitchdowney** [comment]
We are archiving all Github issues and PRs that pertain to the previous generation Podverse. The Podverse dev team is currently 100% focused on the next generation app, and will be only working on next gen Github issues going forward.

If you would like to see a work-in-progress demo of the upcoming Podverse website, please visit https://alpha.podverse.fm. Our tentative release date is Spring 2026.

If you're a developer who may be interested in contributing to Podverse Next-Gen, please see our [Local Dev Setup documentation](https://github.com/podverse/podverse-ops/blob/v5-develop/dev/local-dev-setup.md). We can also be reached through [Matrix or Discord](https://podverse.fm/contact).

I'm sorry for the abrupt announcement, but thank you for your bug reports and feature requests.
